<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QDialog" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#484848;
color:white;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>36</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 36pt &quot;Verdana&quot;;
color: white;
background-color:  #6666ff;</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton#register_button {
     background-color: #8080ff;
	font: 14pt &quot;Tahoma&quot;;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-color: black;
 }
 QPushButton#register_button:pressed {
     background-color: rgb(0, 0, 0);     
 }
 QPushButton#register_button:hover {
     background-color:#6633CC;
 }

</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>201</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: white;
font: 18pt;</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: white;
font: 18pt;</string>
      </property>
      <property name="text">
       <string>Full Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3"/>
  </widget>
  <widget class="QLineEdit" name="email_form">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_form">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="full_name_form">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>269</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>109</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 18pt;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_to_login">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: #d3d3d3;</string>
   </property>
   <property name="text">
    <string>Existing User? Log in</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
